src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx

```diff
@@ -76,18 +76,20 @@ Software built at New Relic goes through these five phases.
 </table>
 
 ## Requirements
-Each project team is assigned a security engineer charged with ensuring the security of New Relic products. In the requirements building phase, the security engineer performs a risk assessment and then adds security requirements for the project. We add privacy and compliance experts to the project teams as needed.
+Each project team for our generally available New Relic One platform is assigned a security engineer charged with reviewing and advision on the security of New Relic products. In the requirements building phase the security engineer performs a risk assessment and then adds security requirements for the project. Privacy and compliance experts are added to the project teams as needed.
 
 ## Design
-During the design phase, the New Relic Security team collaborates with the stakeholders, engineering leaders, and architects to get a detailed shared understanding of the feature. Security engineers at New Relic contribute to the design process with stakeholders by creating a threat model documenting any acceptance criteria, features, or requirements to securely implement the feature. Using the threat model, the security engineer adds detailed specifications for the required controls to the project.
+During the design phase, the New Relic security engineer collaborates with the stakeholders, engineering leaders, and architects to get a detailed shared understanding of the feature. Security engineers at New Relic contribute to the design process with stakeholders by creating a threat model documenting any acceptance criteria, features, or requirements to securely implement the feature. Using the threat model, the security engineer adds  detailed specifications for the required controls to the project.
```

Comment:
@morrisco, looks like an extra space following "adds."

Reply:
Fixed in recent commit